app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
```

Comment:
Let me know if you have any other possible edits before I do `git add --all` and `git commit --am ...` and `git push origin add_knight_piece`. Those are the commands I should do (again), right?